contracts/src/interfaces/IGateway.sol

```diff
@@ -78,6 +78,12 @@ interface IGateway {
     /// @dev Emitted when a command is sent to register a new wrapped token on AssetHub
     event TokenRegistrationSent(address token);
 
+    /// @dev the address of the BEEFY light client used to verify messages.
+    function beefyClient() external view returns (address);
+
+    // @dev the address of the agent executor implementaiton.
+    function agentExecutor() external view returns (address);
+
```

Comment:
Not necessary. If a variable or constant is public it automatically has an accessor generated.